addons/web/static/src/views/graph/graph_view.js

```diff
@@ -72,6 +72,12 @@ export class GraphView extends Component {
     onGraphClicked(domain) {
         const { context, resModel, title } = this.model.metaData;
 
+        Object.keys(context).forEach((x) => {
+            if (x === "group_by" || x.startsWith("search_default_")) {
+                delete context[x];
+            }
+        });
```

Comment:
Similar to what is done for the pivot view [1]. Need to discuss it with the JS framework team

https://github.com/odoo/odoo/blob/bc79a329a1e1edf964f97886943c0688db2087fd/addons/web/static/src/views/pivot/pivot_view.js#L125-L129

Reply:
This is correct. This has been lost in the graph view conversion to owl (see in [legacy](https://github.com/odoo/odoo/blob/15.0/addons/web/static/src/legacy/js/views/graph/graph_controller.js#L332)). This should be tested though.